### DISCUSSION TITLE
Stopped backup in some projects

### DISCUSSION THREAD

**Discussion by PHANTOM-XOTIC** [Sketchware-Pro/Sketchware-Pro#513] [Q&A]
https://user-images.githubusercontent.com/106108205/180776079-429c39ff-0d15-4de1-bce6-5e360149c9d5.mp4

Stopping during project backup Note that only this project cannot be backed up

SketchwarePro_v6.4.0-SNAPSHOT10265bc_minApi26

**Comment by j-px** [comment]
Could you try building any project first, then without closing Sketchware Pro, try to back up that project, and if Sketchware Pro crashes again, upload /Internal storage/.sketchware/debug.txt here?

**Reply by Arashvscode** [reply]
> Could you try building any project first, then without closing Sketchware Pro, try to back up that project, and if Sketchware Pro crashes again, upload /Internal storage/.sketchware/debug.txt here?

Don't worry, his mobile phone has a problem. I tested the last action and it works well
![Screenshot_20220725-173749_Sketchware Pro](https://user-images.githubusercontent.com/101287880/180784807-44fb6382-d6a1-4d02-8cec-9ffd342bcd6a.jpg)

**Reply by PHANTOM-XOTIC** [reply]
I said that only this project cannot be backed up. My phone is , it is Android 9 and it has memory

**Reply by PHANTOM-XOTIC** [reply]
Every project I've ever created has been well backed up Even high volume projects But this project is small in size and maybe 5 MB Why? Now I don't know if the backup is broken Can you recommend a tool for this so I can upload it for you?

**Reply by PHANTOM-XOTIC** [reply]
> > آیا می توانید ابتدا پروژه ای بسازید، سپس بدون بستن Sketchware Pro، سعی کنید از آن پروژه نسخه پشتیبان تهیه کنید و اگر Sketchware Pro دوباره خراب شد، /Internal storage/.sketchware/debug.txt را در اینجا آپلود کنید؟
> 
> نگران نباش موبایلش مشکل داره. من اکشن آخر رو تست کردم و خوب جواب داد ![Screenshot_20220725-173749_Sketchware Pro](https://user-images.githubusercontent.com/101287880/180784807-44fb6382-d6a1-4d02-8cec-9ffd342bcd6a.jpg)

If you think there is a problem with my phone, you are wrong Do not end the problem here
